MenuList.java

```diff
@@ -0,0 +1,62 @@
+import java.util.*;
+/**
+ * This class is used to display the items available in the store
+ * @author Harshit
+ *
+ */
+public class MenuList {
+
+	public static List<Product> menul=new ArrayList<Product>();
+	public static void add()

```

Comment:
Here you can also create a UI for admin where you can provide admin a chance to add or remove items from the menu thus save you from the hard coded values.

Reply:
**Will discuss about this**